activerecord/test/cases/tasks/database_tasks_test.rb

```diff
@@ -8,6 +8,9 @@ def setup
       ActiveRecord::Tasks::MySQLDatabaseTasks.stubs(:new).returns @mysql_tasks
       ActiveRecord::Tasks::PostgreSQLDatabaseTasks.stubs(:new).returns @postgresql_tasks
       ActiveRecord::Tasks::SQLiteDatabaseTasks.stubs(:new).returns @sqlite_tasks
+
+      @original_stdout = $stdout
+      $stdout = StringIO.new
```

Comment:
Are you setting `$stdout` back to `@original_stdout` somewhere (`def teardown …`)?